galsim/chromatic.py

```diff
@@ -131,6 +136,216 @@ def __init__(self, obj):
+        with the best option depending on the application.  For example, when rendering many times
+        with the same achromatic transformation applied, it is typically advantageous to apply the
+        transformation before setting up the interpolation.  But there is no value in this when
+        applying different achromatic transformation to each object.  Chromatic transformations must
+        be applied before setting up interpolation, and attempts to render images of
+        ChromaticObjects with interpolation followed by a chromatic transformation will result in an
+        exception being raised.
+
+        Because of the clever way that the ChromaticConvolution routine works, convolutions of
+        non-separable chromatic objects that use interpolation with separable objects will still
+        benefit from these optimizations.  For example, a non-separable chromatic PSF that uses
+        interpolation, when convolved with a sum of two galaxy components each with their own SED
+        will be able to take advantage of this optimization.  In contrast, when convolving two
+        non-separable profiles there is no way to take advantage of the interpolation optimization,
+        so even in the case that interpolation has been set up, it will be ignored and the full
+        calculation will be done.
+
+        For use cases requiring a high level of precision, we recommend a comparison between the
+        interpolated and the more accurate calculation for at least one case, to ensure that the
+        required precision has been reached.  Also note that after calling `setupInterpolation`, it
+        is possible to revert to the exact calculation by calling `noInterpolation`.
+
+        The input parameter `waves` determines the input grid on which images are precomputed.  It
+        is difficult to give completely general guidance as to how many wavelengths to choose or how
+        they should be spaced; some experimentation compared with the exact calculation is warranted
+        for each particular application.  The best choice of settings might depend on how strongly
+        the parameters of the object depend on wavelength.
+
+        @param waves            The list, tuple, or NumPy array of wavelengths to be used when
+                                building up the grid of images for interpolation.  The wavelengths
+                                should be given in nanometers, and they should span the full range
+                                of wavelengths covered by any bandpass to be used for drawing Images
+                                (i.e., this class will not extrapolate beyond the given range of
+                                wavelengths).  They can be spaced any way the user likes, not
+                                necessarily linearly, though interpolation will be linear in
+                                wavelength.
```

Comment:
Recommend adding "... between the specified wavelengths." at the end of this sentence.